eZ/Publish/Core/Persistence/Legacy/Content/UrlAlias/Handler.php

```diff
@@ -441,6 +441,15 @@ protected function createUrlAlias($action, $path, $forward, $languageCode, $alwa
                 $topElementMD5,
                 $data
             );
+        // adding existing alias to diffrent laquage version
```

Comment:
Please move it below the `elseif` and rephrase:
```php
// add another language to the same custom alias
```
so it reflects more what actually happens.